app/src/main/java/id/rizmaulana/covid19/ui/adapter/viewholders/OverviewItemViewHolder.kt

```diff
@@ -97,6 +96,7 @@ class OverviewItemViewHolder(itemView: View) : BaseViewHolder<OverviewItem>(item
             setDrawEntryLabels(false)
             animateY(PIE_ANIMATION_DURATION, Easing.EaseInOutQuart)
             invalidate()
+            binding.layoutCases.visibility = View.VISIBLE
```

Comment:
You can use view extention
`
binding.layoutCases.visible()
`

Reply:
Okayy @rizmaulana